driver-sync/src/main/com/mongodb/client/ListCollectionsIterable.java

```diff
@@ -27,8 +27,9 @@
  *
  * @param <TResult> The type of the result.
  * @since 3.0
+ * @mongodb.driver.manual reference/command/listCollections/ listCollections
  */
-public interface ListCollectionsIterable<TResult> extends MongoIterable<TResult> {
+public interface ListCollectionsIterable<TResult> extends CommonListCollectionsIterable<TResult> {
```

Comment:
The PR description indicates that `authorizedCollections` method is added to this interface, but it's not.  I don't think it should be, but let's resolve the apparent discrepancy between the description and the code.

I am probably just misinterpreting the description.

Reply:
> The PR description indicates that authorizedCollections method is added to this interface, but it's not.

The PR description says _"Added interfaces/classes with a new method `authorizedCollections`"_. This seems right. The confusion probably comes from the table. I updated it, marked that the existing classes/interfaces were not modified.